src/components/graph/Graph.jsx

```diff
@@ -402,6 +410,17 @@ export default class Graph extends React.Component {
         }
     };
 
+    /**
+     * Handles node position change.
+     * @param {string} nodeId - id of the node whose position changed.
+     * @param {number} x - x coordinate of the node whose position changed.
+     * @param {number} y - y coordinate of the node whose position changed.
+     * @returns {undefined}
+     */
+    onNodePositionChange = (nodeId, x, y) => {
```

Comment:
Please add a sample of this function together with the other callbacks on the `Graph` component jsdoc starting on line 52.

Reply:
Done